spec/services/order_invoice_comparator_spec.rb

```diff
@@ -132,10 +131,8 @@
 
 shared_examples "associated attribute changes - adjustments (update)" do |boolean, type|
   context "adjustment changes" do
-    let!(:adjustment) { create(:adjustment, order_id: order.id) }
-
     context "with an existing adjustment" do
-      before { adjustment.update!(label: "It's a new label") }
+      before { Spree::Adjustment.first.update!(label: "It's a new label") }
```

Comment:
It looks like this is selecting the first adjustment that was automatically created as part of the order. Could we make that clearer with something like:

```suggestion
      before { order.adjustments.first.update!(label: "It's a new label") }
```

Reply:
Makes sense, that would make it more clear. However - and I'm not sure why:

```
order.adjustments.count
=> 0
```

adjustments comes from `/app/services/order_adjustments_fetcher.rb:38`

```
order.all_adjustments.count
=> 2
```

So, I guess, we would have to do:

`before { order.all_adjustments.first.update!(label: "It's a new label") }`

Reply:
That is odd.. but looks like a good solution, well done 👍